src/execute/ExecuteNotesMode.tsx

```diff
@@ -0,0 +1,204 @@
+import { Button, Box, Text } from "@chakra-ui/react";
+import { GrowthyConversation } from "common/components/GrowthyConversation";
+import { Outline } from "common/components/outline/Outline";
+import { UIBlogArticle } from "domain/blog-article/UIBlogArticle";
+import { Notes } from "notes/Notes";
+import { FaArrowLeft, FaArrowRight } from "react-icons/fa";
+import { Actions } from "./Actions";
+import { AIGenerateContent } from "./growthy-ai-panel/AIGenerateContent";
+import { UIOutline } from "domain/common/UIOutline";
+import { useRef, useState } from "react";
+import { getGuidance as defaultGetGuidance } from "execute/chains/getGuidance";
+
+type Props = {
+  exerciseXML: string;
+  growthExerciseId: string | undefined;
+  handleOutlineUpdate: (uiOutline: UIOutline) => void;
+  getGuidance?: (
+    inputs: {
+      blog_article_goal: string;
+      blog_article_xml: string;
+      blog_article_task: string;
+    },
+    context: string,
+    isInitialPrompt: boolean,
+    conversation: { type: string; text: string }[]
+  ) => void;
+};
+
+export const ExecuteNotesMode: React.FC<Props> = ({
```

Comment:
What dont we name this NotesMode instead, any specific reason to prepend Execute

Reply:
Done